chapter_natural-language-processing-pretraining/bert-dataset_vn.md

```diff
@@ -19,15 +19,20 @@ Thus, it is getting popular to pretrain BERT on a customized dataset.
 To facilitate the demonstration of BERT pretraining, we use a smaller corpus WikiText-2 :cite:`Merity.Xiong.Bradbury.ea.2016`.
 -->
 
-*dịch đoạn phía trên*
+Để tiền huấn luyện mô hình BERT như thực hiện trong :numref:`sec_bert`, ta cần sinh tập dữ liệu ở định dạng lý tưởng để thuận tiện cho hai tác vụ tiền huấn luyện: dựng mô hình   từ ẩn và dự đoán câu kế.
+Một mặt, mô hình BERT gốc được tiền huấn luyện trên sự ghép nối của hai kho ngữ liệu khổng lồ BookCorpus và Wikipedia Tiếng Anh (xem :numref:`subsec_bert_pretraining_tasks`), làm khó vận hành đối với hầu hết máy đọc cuốn sách này.
```

Comment:
```suggestion
Một mặt, mô hình BERT gốc được tiền huấn luyện trên kho ngữ liệu được ghép lại từ bộ BookCorpus khổng lồ và Wikipedia Tiếng Anh (xem :numref:`subsec_bert_pretraining_tasks`), gây khó khăn khi thực hành đối với hầu hết bạn đọc cuốn sách này.
```

Reply:
Chỗ này mình không rõ vì sao lại là người đọc, reader ở đây mình nghĩ là ám chỉ máy đọc.

Reply:
Ý của câu trên là vì kho ngữ liệu quá khổng lồ nên người đọc cuốn sách này sẽ không có đủ tài nguyên tính toán để chạy tiền huấn luyện BERT